### ISSUE TITLE
Errors using "CORNERS" mode 

### ISSUE THREAD

**Issue by wallygit** [p5py/p5#438]
I get an error message when I try to use the "CORNERS" mode in p5.rect().

```
File "path?p5/issue_example_code.py", line 9, in draw
    rect(100, 100, 50, 50)
  File path/p5/.p5venv/lib/python3.10/site-packages/p5/core/primitives.py", line 605, in rect
    (corner_2,) = args
ValueError: too many values to unpack (expected 1)
```

```
from p5 import *
def setup():
    size(400, 300)
def draw():
    background(51)

    rectMode(CORNERS)
    fill(0, 255, 0)
    rect(100, 100, 50, 50)
   
run(renderer="vispy")
```

**Expected behavior**
draw a rectangle

**System information:**
 - p5 release 0.82
 - Python version: 3.10
 - Operating system: Kubuntu 22.0.4

**Additional context**

**Comment by hvitis** [comment]
Opened PR for this. @tushar5526  Let me know what else can I do in order to merge it.

**Comment by tushar5526** [comment]
Hi @hvitis, will take a look in few days. Appreciate the PR :)

**Comment by hvitis** [comment]
Thank you!

sob., 30 wrz 2023 o 19:09 Tushar ***@***.***> napisał(a):

**Comment by tushar5526** [comment]
Closed by #442. Thank you @hvitis
